migrations/20230312043023_user.sql

```diff
@@ -3,9 +3,9 @@
 CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
 -- user table
 CREATE TABLE IF NOT EXISTS af_user (
-    uid BIGSERIAL PRIMARY KEY,
-    email TEXT NOT NULL DEFAULT '' UNIQUE,
-    password TEXT NOT NULL,
+    uid UUID PRIMARY KEY, -- related to gotrue user when authenticated
```

Comment:
The uid must be big int. I used it in appflowy applicaiton. How about create a new column: uuid

Reply:
I've made the changes to use the existing BIGINT for user id